docs/source/UIDocumentation.md

```diff
@@ -1,160 +1,238 @@
+* ```addNewLayer``` - invoked by ```handleClick``` and passed in a JS object with layer information - adds a layer
 * ```changeSelectedLayer``` - changes which layer has the selected class on it, which outlines layer to emphasize a "selection"
-  * layerId
 * ```changeHoveredLayer``` - changes which layer has the hover class on it, which outlines layer to emphasize a "hover"
-  * layerId
-* ```modifyLayer``` - modifies layer, passed in layer is the new layer, and layer id is the id it needs to be replaced at. 
-  * layer
-  * layerId
-* ```modifyLayerParams``` - modify layer params based on layer and layerId, invoked by setParams
-  * layer
-  * layerId
-* ```deleteLayer``` - deletes layers and removes inputs and outputs for it. 
-  * layerId
-* ```updateParameters``` - sums total amount of parameters and updates layer's parameters
-  * layer
-  * net
-* ```calculateParameters``` - loops through net and invokes ```updateParameters```
-  * net
+* ```addHighlightOnLayer``` - highlights layer by ```layerId```
+* ```modifyLayer``` - modifies layer, passed in layer is the new layer, and layer id is the id it needs to be replaced at.
+* ```deleteLayer``` - deletes layers and removes inputs and outputs for it.
+* ```getLayerParameters``` - sums total amount of parameters and updates layer's parameters
+* ```calculateParameters``` - loops through net and invokes ```getLayerParameters```
+* ```adjustParameters``` - used to adjust layer parameters based on layer, change is passed in.
+* ```modifyLayerParams``` - modifies layer params based on layer and layerId, invoked by SetParams
 * ```loadLayerShapes``` - AJAXs to backend to model parameters
 * ```exportNet``` - AJAXs to backend and then passes back error/success
-  * framework
+* ```exportPrep``` - invoked by ```exportNet```; prepares for model export
 * ```importNet``` - AJAXs to backend and then passes back error/success
-  * framework
-  * id
 * ```initialiseImportedNet``` - starts prepping layer to be displayed by Fabrik, positions layers
-  * net
-  * net_name
-* ```changeNetName``` - invoked on the event of net name changed
-  * event
-* ```adjustParameters``` - used to adjust layer parameters based on layer, change is passed in. 
-  * layer
-  * para
-  * value
-* ```changeNetStatus``` - changes boolean for rebuilding. 
-  * bool
-* ```changeNetPhase```
-  * phase
-* ```dismissError```
-  * errorIndex
-* ```addError```
-  * errorText
-* ```dismissAllErrors```
-* ```copyTrain``` - copy's train net for test
+* ```changeNetName``` - invoked on the event of net name changed; changes the name of the net
+* ```changeNetStatus``` - changes boolean for rebuilding.
+* ```changeNetPhase``` - changes the phase of the net
+* ```dismissError``` - dissmisses the error by ```errorIndex```
+* ```addError``` - adds error using ```errorText```
+* ```dismissAllErrors``` - dismisses all errors
+* ```copyTrain``` - copies the nets train option for the test option
 * ```trainOnly```
```

Comment:
No explanation for this method? Having no explanation looks a bit weird while all other methods have them. Either move this method to the end of the list, or provide a brief explanation.